scripts/arangodb-bash-setup/test-data/testData.json

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
We do not want a second copy of this. Be sure to delete or git-ignore it. In fact,  I wouldn't `curl` it to anywhere within the repo- it will cause problems with the diff.